build.zig

```diff
@@ -64,9 +92,11 @@ pub fn build(b: *Builder) !void {
     exec.setBuildMode(build_mode);
     exec.setLinkerScriptPath(linker_script_path);
     exec.setTarget(target);
+    exec.code_model = .kernel;
 
     const make_iso = switch (target.getCpuArch()) {
         .i386 => b.addSystemCommand(&[_][]const u8{ "./makeiso.sh", boot_path, modules_path, iso_dir_path, exec.getOutputPath(), ramdisk_path, output_iso }),
+        .x86_64 => b.addSystemCommand(&[_][]const u8{ "./makeiso_64.sh", b.exe_dir, exec.getOutputPath(), output_iso, ramdisk_path }),
```

Comment:
Would it be feasible to have just one makiso.sh file and change behaviour based on the architecture being built?

Reply:
The two files don't have much in common, so is easier to have 2 separate bash files than switching inside the bash file for the build target. This might change later on though.